packages/core/src/BaseDropdown/DropdownPanel.tsx

```diff
@@ -0,0 +1,89 @@
+import { forwardRef } from "react";
+import { theme } from "@hitachivantara/uikit-styles";
+
+import { HvPanel, HvPanelProps } from "../Panel";
+import { createClasses, ExtractNames } from "../utils/classes";
+
+export const { staticClasses, useClasses } = createClasses("HvDropdownPanel", {
+  panel: {
+    border: `1px solid ${theme.colors.secondary}`,
+    // borderRadius: 0,
+    marginTop: -1,
+    marginBottom: -1,
+  },
+  panelOpenedUp: {
+    borderRadius: `${theme.radii.base} ${theme.radii.base} 0 0`,
+  },
+  panelOpenedDown: {
+    borderRadius: `0 0 ${theme.radii.base} ${theme.radii.base}`,
+  },
+  extensionOpen: {
+    height: 0,
+    backgroundColor: theme.colors.atmo1,
+    borderTop: "none",
+    borderBottom: "none",
+    borderRight: `1px solid ${theme.colors.secondary}`,
+    borderLeft: `1px solid ${theme.colors.secondary}`,
+    boxShadow: `0px 8px 0px ${theme.colors.atmo1}, 0px 0px 9px 0px rgba(65,65,65,.12)`,
+  },
+  extensionLeftPosition: { marginLeft: "auto" },
+  extensionFloatRight: { float: "right" },
+  extensionFloatLeft: { float: "left" },
+});
+
+export type HvDropdownPanelClasses = ExtractNames<typeof useClasses>;
+
+export interface HvDropdownPanelProps extends Omit<HvPanelProps, "classes"> {
+  classes?: HvDropdownPanelClasses;
+  placement?: string;
+}
+
+/**
+ * HvBaseDropdown container HvPanel that includes DS3 "extensions"
+ *
+ * TODO:
+ * - make default HvBaseDropdown inherit `HvPanel`'s padding
+ * - refactor extensions into ::before and ::after to remove DS3-only DOM elements
+ */
+export const HvDropdownPanel = forwardRef<HTMLDivElement, HvDropdownPanelProps>(
```

Comment:
I noticed the following in DS3:

https://github.com/lumada-design/hv-uikit-react/assets/43220251/9f5e2d27-8042-4ca5-8656-b7a129585faf

Maybe this has something to do with the `extensionWidth` that was removed in the base dropdown component. 

I also noticed a visual change in the extension regarding the box shadow. Looking at what we previously had, the box shadow seemed less pronounced.